src/components/overlays/NewNotifications/NotificationView/NotificationView.tsx

```diff
@@ -105,7 +95,7 @@ export const NotificationView = forwardRef(function NotificationView(
         >
           <NotificationIcon icon={icon} type={type} />
 
-          {header && <NotificationHeader header={header} id={labelID} />}
+          <NotificationHeader header={header} id={labelID} />
```

Comment:
According to the design, the header is not required. But I think you faced some limitations. If so, just ignore this comment.